### ISSUE TITLE
Backend session expiration

### ISSUE THREAD

**Issue by mosid** [octobercms/october#2394]
##### Expected behavior

Backend session expires when setting the 'expire_on_close' to true or setting lifetime to zero or whatever.
##### Actual behavior

Session options 'expire_on_close' and 'lifetime' are bypassed even after setting them. session never expires.
##### Reproduce steps
- open `session.php` in config folder
- set `expire_on_close=>true` and `lifetime=>0`  
##### October build

363

**Comment by Nyorai** [comment]
I've come across this and was trying to find a solution but this behaviour is hardcoded into `October\Rain\Auth\Manager` and `Backend\Controllers\Auth`. The controller calls the `authenticate` method of the manager with `$remember` set to `true`, which then creates a forever cookie in the `login` method (line 420).

```
if ($remember) {
    Cookie::queue(Cookie::forever($this->sessionKey, $toPersist));
}
```

The `expire_on_close` and `lifetime` settings only apply to the `october_session` cookie, not the `admin_auth` cookie which is used on the backend or the `user_auth` which is user by the Rainlab User plugin.

**Comment by Nyorai** [comment]
Forgive me, I'm not fully up to speed with Github and pull-requests, but I've made some changes to the backend module to deal with this issue.

https://github.com/Nyorai/october/pull/1

**Comment by daftspunk** [comment]
I'm seeing two different issues here.  This issue lacks enough detail or the replication instructions cannot be actioned easily. As per [the contribution guidelines](https://github.com/octobercms/october/blob/master/CONTRIBUTING.md#reporting-a-bug-with-octobercms), please provide the exact steps you are taking for the issue to occur. This will be used for peer review.

Here is an example of a **bug summary**:

> The CMS page URL is not being set automatically from the title value.

Here is an example of some **reproduce steps**:

> 1. In the back-end, open the CMS > Pages section
> 1. Click the "Add" button
> 1. Enter [Hello] for the Title field
> 1. Click the Save button 

Here is an example of **expected behavior**:

> - The Title should be set to [Hello]
> - The URL should be set to [/hello]

Here is an example of **actual behavior**:

> - The Title is set to [Hello] correctly
> - The URL is left empty

**Comment by LukeTowers** [comment]
Fixed by #2924
